.github/workflows/build_and_test.yaml

```diff
@@ -2,172 +2,167 @@ name: Build and Test
 on:
   push:
     branches:
-    - "main"
-    - "release/v*"
+      - "main"
```

Comment:
can we undo all changes from this file  ?
the existing indentation seems consistent with our code base

Reply:
that won't pass the yaml lint.

Reply:
Not seeing any issues when Im running yamllint

Reply:
you didn't change it?

Reply:
updated